lib/java/com/google/android/material/drawable/DrawableUtils.java

```diff
@@ -205,44 +228,51 @@ public static Drawable compositeTwoLayeredDrawable(
     }
     LayerDrawable drawable =
         new LayerDrawable(new Drawable[] {bottomLayerDrawable, topLayerDrawable});
+
+    final int topLayerWidth = topLayerDrawable.getIntrinsicWidth();
```

Comment:
For clarity, I suggest renaming to topLayerPreviousWidth. Same for the other three new width/height variables.

Reply:
I think `***PreviousWidth` is even worse, but I think I get what you mean. How about `topLayerOriginalWidth`?

Reply:
`topLayerOriginalWidth` sounds great to me.

Reply:
Good.